tests/e2e/query.go

```diff
@@ -170,3 +171,40 @@ func queryDelegatorWithdrawalAddress(endpoint string, delegatorAddr string) (dis
 	}
 	return res, nil
 }
+
+func queryValidator(endpoint, address string) (stakingtypes.Validator, error) {
+	var res stakingtypes.QueryValidatorResponse
+
+	resp, err := http.Get(fmt.Sprintf("%s/cosmos/staking/v1beta1/validators/%s", endpoint, address))
```

Comment:
Up to you but perhaps we could start to use [this](https://github.com/cosmos/gaia/blob/main/tests/e2e/http_util.go#L9) more in our REST queries